hack/swift/Makefile

```diff
@@ -9,6 +9,7 @@ AZCLI   ?= docker run -v $(AZCFG):/root/.azure -v $(KUBECFG):/root/.kube -v $(SS
 
 # overrideable variables
 USER       ?= $(whoami)
+OSSKU      ?= Ubuntu
```

Comment:
is this case sensitive?

Reply:
Not sure, will test it out though. The documentation had this casing so went with that - https://learn.microsoft.com/en-us/cli/azure/aks?view=azure-cli-latest#az-aks-create